liquibase-standard/src/test/groovy/liquibase/datatype/DataTypeFactoryTest.groovy

```diff
@@ -212,6 +212,7 @@ class DataTypeFactoryTest extends Specification {
         "mediumblob"                                   | new MySQLDatabase()    | "MEDIUMBLOB"                                   | BlobType      | false
         "mediumtext"                                   | new MySQLDatabase()    | "MEDIUMTEXT"                                   | ClobType      | false
         "real"                                         | new MySQLDatabase()    | "REAL"                                         | FloatType     | false
+        "uuid"                                         | new MySQLDatabase()    | "binary(16)"                                   | UUIDType      | false
```

Comment:
This should be BINARY uppercase. When tests are run, `getDatabaseMajorVersion` is called from `AbstractJdbcDatabase` returning `999` .